functions/Get-DbaUserPermission.ps1

```diff
@@ -240,23 +288,16 @@ function Get-DbaUserPermission {
                     $null = 1
```

Comment:
Again: Why don't you care?

Reply:
See comment above: this was working before I started making any changes and is not related to problem I am trying to solve with my update.